developer/architecture/permissions.md

```diff
@@ -50,6 +50,9 @@ Reaction.registerPackage({
     }]
    });
 ```
+At the point where Packages are published in the app, each registry item permissions are collected and put on the
+package registry [(source)](https://github.com/reactioncommerce/reaction/blob/master/server/publications/collections/packages.js#L31-L56).
+Based on these permissions, restriction to certain functionalities can then be granted or prevented depending on user roles.
```

Comment:
Not sure how the logic of this sentence works: "restrictions to certain functionalities can be granted". So are we granting restrictions? Maybe something like "Based on these permissions we can enable or disable functionality depending on user roles"?